Exceptions/InternalException.php

```diff
@@ -18,17 +17,31 @@ class InternalException extends \Exception implements HttpExceptionInterface
     protected $statusCode;
 
     /**
-     * BaseInternalException constructor.
+     * @var array
+     */
+    protected $body;
+
+    /**
+     * @var int
+     */
+    protected $errorCode;
+
+    /**
+     * InternalException constructor.
      * @param string $message
-     * @param int $statusCode
+     * @param $statusCode
+     * @param int $errorCode
      * @param array $headers
-     * @param null|Throwable $previous
+     * @param array $body
+     * @param Throwable|null $previous
      * @param int $code
      */
-    public function __construct($message = '', $statusCode, $headers = [], Throwable $previous = null, $code = 0)
+    public function __construct($message = '', $statusCode, $errorCode = self::DEFAULT_ERROR_CODE, $headers = [], $body = [],  Throwable $previous = null, $code = 0)
```

Comment:
Not convenient sequence. Let's think over common cases.
A developer will create InernalExeption(message, errorCode, additionalData)
When do you stand on this?

Reply:
Your suggestion?